azure-pipelines.yml

```diff
@@ -9,7 +9,7 @@ trigger:
 pool:
   vmImage: 'ubuntu-latest'
```

Comment:
can we use image windows-latest ?
https://docs.microsoft.com/ru-ru/azure/devops/pipelines/agents/hosted?view=azure-devops

it has pre-installed software: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/win/Vs2019-Server2019-Readme.md